.drone.star

```diff
@@ -137,8 +132,8 @@ def repo(name, url = "", git = "", sub_path = "", branch = "master", mode = "mak
                 },
                 "commands": [
                     "cd '%s'" % work_dir,
-                    "npm install --silent --global --force \"$(jq -r '.packageManager' < package.json)\";" + \
-                    "pnpm config set store-dir ./.pnpm-store;" + \
+                    "npm install --silent --global --force \"$(jq -r '.packageManager' < package.json)\"",
+                    "pnpm config set store-dir ./.pnpm-store",
```

Comment:
This change seems to have broken CI
https://drone.owncloud.com/owncloud/translation-sync/3157/1/4
```
+ cd 'core/l10n'
+ npm install --silent --global --force "$(jq -r '.packageManager' < package.json)"
/bin/sh: 42: cannot open package.json: No such file
```
@saw-jan why did this need to be changed?

Reply:
I though these commands are for all. I didn't see the if block.